src/modules/github.cpp

```diff
@@ -25,21 +25,25 @@ namespace modules {
     if (m_formatter->has(TAG_LABEL)) {
       m_label = load_optional_label(m_conf, name(), TAG_LABEL, "Notifications: %notifications%");
 
-      if (m_empty_notifications) {
-        m_label->replace_token("%notifications%", "0");
-      } else {
-        m_label->clear();
-      }
+      update_label(-1);
```

Comment:
I think we should keep the behaviour that the module is initialized with 0

Reply:
Actually it is confusing for me to see no notification, when actually no information is present.
So I preferred to display other value then 0. 
I am planning to introduce the `label-offline` which would be the best option so I changed back to 0 for now :)